src/java.desktop/macosx/classes/com/apple/laf/AquaScrollBarUI.java

```diff
@@ -527,6 +527,12 @@ void setScrollByBlock(final boolean block) {
         }
 
         public void actionPerformed(final ActionEvent e) {
+            if (fTrackHighlight != Hit.NONE && !fTrackListener.fStillInTrack) {
```

Comment:
`fTrackListener.fStillInTrack` value is `true` and here the condition is to check for negation which makes it to `false` and overall condition evaluates to `false` and that may be the reason that the timer is not stopped.

Reply:
Yes thats the drawback of indirect checking....I have modified PR to check for frame disable directly and stop the timer..